packages/components/src/components/Select/Select.tsx

```diff
@@ -9,12 +9,16 @@ import {
 } from "ariakit/select";
 import type { SelectProps as SelectPrimitiveProps } from "ariakit/select";
 import type { SystemProp, Theme } from "@xstyled/styled-components";
+import isArray from "lodash/isArray";
 import { Box } from "../../primitives/Box";
+import { Text } from "../../primitives/Text";
 import { Icon } from "../Icon";
 import { SelectItem } from "./SelectItem";
 import type { SelectItemProps } from "./SelectItem";
 import { SelectPopover } from "./SelectPopover";
 
+const NO_SELECTION_STRING = "Select one";
```

Comment:
Should this be override-able?

Reply:
Probably a good call.

Reply:
Added an optional `placeholder` prop that can replace that default string.